packages/ibm-products/src/components/TagOverflow/TagOverflowPopover.tsx

```diff
@@ -8,7 +8,14 @@
 import React, { useRef, forwardRef, Ref } from 'react';
 import PropTypes from 'prop-types';
 import cx from 'classnames';
-import { Link, Tag, Popover, PopoverContent } from '@carbon/react';
+import {
+  Link,
+  Tag,
+  Popover,
+  PopoverContent,
+  /**@ts-ignore */
```

Comment:
you can alleviate the import error by including the import [here](https://github.com/carbon-design-system/ibm-products/blob/main/packages/ibm-products/src/custom-typings/index.d.ts)

Reply:
Thanks @davidmenendez!
This is updated now.